pkg/tfgen/main.go

```diff
@@ -116,16 +116,20 @@ func newTFGenCmd(pkg string, version string, prov tfbridge.ProviderInfo) *cobra.
 				root = afero.NewBasePathFs(afero.NewOsFs(), absOutDir)
 			}
 
+			// Creating an item to keep track of example coverage
+			var coverageTracker CoverageTracker = newCoverageTracker(prov.Name)
```

Comment:
I don't think we should instantiate a coverageTracker if there is not the requirement to do so - i.e. an env var suggesting that you want an output

Thoughts?

Reply:
Changed coverage tracker to not be instantiated if environment variable isn't present.